test/integration/taskBasedStatusUpdate.test.js

```diff
@@ -282,4 +284,76 @@ describe("Task Based Status Updates", function () {
       );
     });
   });
+
+  describe("GET users/status?taskStatus=IDLE Find Users without Assigned Or InProgress Tasks", function () {
+    let userId1;
+    let userId2;
+    let userId3;
+
+    let superUserId;
+    let superUserJwt;
+
+    beforeEach(async function () {
+      userId1 = await addUser(userData[6]);
+      userId2 = await addUser(userData[8]);
+      userId3 = await addUser(userData[9]);
+      superUserId = await addUser(userData[4]);
+      superUserJwt = authService.generateAuthToken({ userId: superUserId });
+
+      const taskArr = allTasks();
+
+      const sampleTask1 = taskArr[0];
+      sampleTask1.assignee = userId1;
+      sampleTask1.status = "ASSIGNED";
```

Comment:
why you are changing sampleTask1.status from "PROGRESS" to "ASSIGNED"

Reply:
Because in the fixture the taskArr[0] has the status IN_PROGRESS. I am overwriting that with ASSIGNED